### PR TITLE
{bio}[foss-2015b] SAMtools, HTSlib and BCFtools v1.4.1 (REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**

### CODE REVIEW

File: easybuild/easyconfigs/b/BCFtools/BCFtools-1.4.1-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['https://github.com/samtools/bcftools/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [
+	('zlib', '1.2.8'),
+	('GSL', '2.1'),
+        ('bzip2', '1.0.6'),
+        ('XZ', '5.2.2'),
```
Comment (boegel) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.4.1-foss-2015b.eb]: please fix indent (4 spaces)

File: easybuild/easyconfigs/b/BCFtools/BCFtools-1.4.1-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+buildopts = " USE_GPL=1 GSL_LIBS='-lgsl -lgslcblas'"
+
+runtest = 'test'
+
+files_to_copy = [(["bcftools", "misc/*.*"], "bin"),
+		   "doc", "plugins", "test", "LICENSE", "README", "AUTHORS"]
+
+sanity_check_paths = {
+    'files': ["bin/bcftools"],
```
Comment (boegel) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.4.1-foss-2015b.eb]: what other stuff is supposed to be copied from `misc`?

File: easybuild/easyconfigs/s/SAMtools/SAMtools-1.4.1-foss-2015b.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+sources = [SOURCELOWER_TAR_BZ2]
+
+# The htslib component of SAMtools 1.4 uses zlib, bzip2 and lzma compression.
+# The latter is currently provided by XZ.
+dependencies = [
+    ('ncurses', '5.9'),
+    ('zlib',    '1.2.8'),
+    ('bzip2',   '1.0.6'),
+    ('XZ',      '5.2.2'),
+    ('cURL',    '7.45.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/s/SAMtools/SAMtools-1.4.1-foss-2015b.eb]: we generally don't align dep versions like this (comparing these versions is meaningless, so no need to align them), just stick to a single space after `,`?

**Comment by boegel** [comment]
@nathanhaigh ping on remarks & missing dep?

**Comment by migueldiascosta** [comment]
@nathanhaigh, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**